test/tablet.py

```diff
@@ -563,11 +563,11 @@ def start_vttablet(
 
     return self.proc
 
-  def wait_for_vttablet_state(self, expected, timeout=60.0, port=None):
+  def _wait_for_vttablet_var(self, var_name, expected, timeout=60.0, port=None):
```

Comment:
Note that we already have `poll_for_vars` in `test/utils.py`.

After your change, this function sounds to be very similar to the other one? Maybe merge them?
(`poll_for_vars` is mostly used in `worker.py`.)